Assets/MRTK/SDK/Features/UX/Prefabs/Dialog/DialogMedium_192x128.prefab

```diff
@@ -9,13 +9,13 @@ PrefabInstance:
     m_Modifications:
     - target: {fileID: 482541237737822506, guid: 478732197b53dee4694480aac1a7c98c,
         type: 3}
-      propertyPath: m_textInfo.characterCount
-      value: 18
+      propertyPath: m_textInfo.lineCount
+      value: 1
       objectReference: {fileID: 0}
```

Comment:
This is not what I see in Unity 2019.4.20. Maybe a serialization issue.
Originally the rearrangement of those buttons happened due to the new OnValidate based layouting in BaseObjectCollection. That broke one of the unit tests. Since these dialogs have fixed button positions (either 1 or 2 active at once) there should not be a GridObjectCollection on them that arranges all three of them. I removed the GridObjectCollections on these prefabs, but apparently the button arrangement was still influenced when doing that.
I will check what these prefabs look like in Unity 2018.

Reply:
I opened these dialog prefabs in Unity 2018.4.32 and they look fine. Is this happening in an example scene or when opening the prefab?

Reply:
This was happening in the example scene.  Since adding OnValidate in BaseGridObjectCollection has an impact on existing prefabs, can we instead update the the referenced collection in your GridObjectCollectionSizeFitter script?  This script has [ExecuteInEditMode] so it is possible update a referenced collection in Update().